v3/lints/cabf_br/lint_crl_next_update_invalid_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * ZLint Copyright 2024 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package cabf_br
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
+/*
+ * Explanation of test file names:
+ *
+ *      nup(1|0) = CRL nextUpdate is present (1) or absent (0)
+ *      sub(1|0) = CRL covers (1) Subscriber certificates or not (0)
+ *      len(1|0) = CRL lifespan is within (0) or beyond (1) the limit set by BRs
+ *      eff(1|0) = CRL thisUpdate is before (0) this lint's effective date or after it (1)
+ */
+
+func TestCrlNextUpdateInvalid(t *testing.T) {
+
+	type Data struct {
+		input  string
+		config string
+		want   lint.LintStatus
+	}
+	data := []Data{
```

Comment:
As always, thank you for the comprehensive test suite. These check out against my own expectations.